src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/PuppetGatlingBuildAction.java

```diff
@@ -1,10 +1,14 @@
-package com.puppetlabs.jenkins.plugins.puppetgatling.gatling;
+package com.puppetlabs.jenkins.plugins.puppetgatling;
 
 import static com.puppetlabs.jenkins.plugins.puppetgatling.Constant.*;
 
 import java.util.*;
 
+import com.puppetlabs.jenkins.plugins.puppetgatling.SimulationMetrics;
 import com.puppetlabs.jenkins.plugins.puppetgatling.PuppetGatlingProjectAction;
+import com.puppetlabs.jenkins.plugins.puppetgatling.chart.Graph;
+import com.puppetlabs.jenkins.plugins.puppetgatling.gatling.ReportRenderer;
+import com.puppetlabs.jenkins.plugins.puppetgatling.gatling.SimulationReport;
```

Comment:
These are all just an artifact of me having moved some classes around to better packages, sorry.